redesign.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 
   content: [
     "./app/assets/**/*.svg",
-    "./app/components/**/*.{rb,html,html.erb,yml}",
+    "./app/components/**/*.{rb,html,html.erb,html+redesign.erb,yml}",
```

Comment:
These changes ensure other variants get picked up, too.

```suggestion
    "./app/components/**/*.{rb,html,yml}?(+*)?(.erb)",
```

And the following:

```diff
-    "./app/views/**/*.html.erb",
-    "./app/views/**/*.html+redesign.erb",
+    "./app/views/**/*.html?(+*).erb",
```

Reply:
Good thinking. I can pop this over into the other thread too.